test/db/index.js

```diff
@@ -312,7 +321,7 @@ describe('db', function() {
         return db.getAccessToken(token);
       }).then(function (t) {
         assert.equal(t, undefined, 'token deleted');
-        return db.getRefreshToken(refreshToken);
+        return db.getRefreshToken(encrypt.hash(refreshToken));
       }).then(function (t) {
         assert.equal(t, undefined, 'refresh_token deleted');
```

Comment:
@seanmonstar without the change above this is always undefined, even if the user was not deleted.